src/components/LineChart.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Line } from 'react-chartjs-2';
+import { Col, Row, Typography } from 'antd';
+
+const { Title } = Typography;
+
+const LineChart = ({ coinHistory, currentPrice, coinName }) => {
+    const coinPrice = [];
+    const coinTimestamp = [];
+
+    for(let i = 0; i < coinHistory?.data?.history?.length; i += 1) {
+        coinPrice.push(coinHistory.data?.history[i].price);   
+        coinTimestamp.push(new Date(coinHistory.data?.history[i].timestamp).toLocaleDateString());
+    };
+
+    const data = {
+        labels: coinTimestamp,
+        datasets: [
+            {
+                label: 'Price in USD',
+                data: coinPrice,
+                fill: false,
+                backgroundColor: '#0071bd',
+                borderColor: '#0071bd'
```

Comment:
Confused here? Your dataset probably doesn't need to contain colors. Maybe move to the component that displays the dataset

Reply:
This was where my whole issue came up (this was a 2.9.4 format).  the 'data: coinPrice' and the "labels: coinTimestamp" are the only data items that i brought into the chart.   The colors and fill, rest are the arbitrary values that I selected. 
Once I am done with this test, I am going to reinstall the newer chartJS back and try some new charts for the next round of coding...